SU-SWS/stanford_events
# D8CORE-000 Event card views stack on smaller screens.

# READY FOR REVIEW

# Summary
- make the card list displays stack when on smaller screens.

# Need Review By (Date)
- 10/3

# Urgency
- medium

# Steps to Test
1. checkout this branch and clear browser caches
1. create a basic page with a list paragraph type and use the Events: "Cards" list as a single item in a row.
1. ensure on smaller screens the items stack.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
